City-of-Helsinki/hel-fi-drupal-grants
# UHF-10455: Add new General grants for welfare and health promotion form

# [UHF-10455](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10455)
Add new form according to spec.

## What was done
* Add new form configs
* Add new `grants_metadata` objects

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout origin/UHF-10455`
  * `make fresh`
* Run `make drush-cr`

## How to test

* Open up new form: http://hel-fi-drupal-grant-applications.docker.so/fi/uusi-hakemus/hyteed_yleis
* Follow the Suomi.fi process
* You are redirected out of the form after logging in, use the link again to get back to the form
* Fill in the form. Intermittently use the "Save as a draft" button and see if it works.
* Submit the form.
  * Wait of the application status to update to "Received"
  * Click on the "View application" button.
  * Try sending some messages for good measure. 
  * Verify that data matches what you tried to send.

## Designers review




## Automatic- / Regression tests



[UHF-10455]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10455?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
